<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>738</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>461</width>
      <height>621</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="xeDoubleSpinBox">
       <property name="minimum">
        <double>-4.000000000000000</double>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="bgColorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="phixTextLabel">
       <property name="text">
        <string>φ&lt;sub&gt;x&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="dzTextLabel">
       <property name="text">
        <string>dz</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="8">
      <widget class="QPushButton" name="clearAllPushButton">
       <property name="text">
        <string>Очистить холст</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="dxTextLabel">
       <property name="text">
        <string>dx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="xbTextLabel">
       <property name="text">
        <string>X&lt;sub&gt;b&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="7">
      <widget class="QPushButton" name="selectBgColorPushButton">
       <property name="text">
        <string>Изменить цвет фона</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zbDoubleSpinBox">
       <property name="minimum">
        <double>-4.000000000000000</double>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="value">
        <double>-2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="8">
      <widget class="Line" name="line3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QDoubleSpinBox" name="dxDoubleSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="fgColorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="8">
      <widget class="Line" name="line6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="phizTextLabel">
       <property name="text">
        <string>φ&lt;sub&gt;z&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="zbTextLabel">
       <property name="text">
        <string>z&lt;sub&gt;b&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="Line" name="line5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QDoubleSpinBox" name="phizDoubleSpinBox">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="2">
      <widget class="Line" name="line2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="8">
      <widget class="QPushButton" name="plotPushButton">
       <property name="text">
        <string>Отобразить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QDoubleSpinBox" name="dzDoubleSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="xeTextLabel">
       <property name="text">
        <string>X&lt;sub&gt;e&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="zeTextLabel">
       <property name="text">
        <string>z&lt;sub&gt;e&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="phiyTextLabel">
       <property name="text">
        <string>φ&lt;sub&gt;y&lt;/sub&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="zeDoubleSpinBox">
       <property name="minimum">
        <double>-4.000000000000000</double>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QDoubleSpinBox" name="phiyDoubleSpinBox">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="8">
      <widget class="Line" name="line7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="Line" name="line4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="7">
      <widget class="QPushButton" name="selectFgColorPushButton">
       <property name="text">
        <string>Изменить цвет линий</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="phixDoubleSpinBox">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="8">
      <widget class="QPushButton" name="rotatePushButton">
       <property name="text">
        <string>Вращать</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <widget class="Line" name="line1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xbDoubleSpinBox">
       <property name="minimum">
        <double>-4.000000000000000</double>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="value">
        <double>-2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="QComboBox" name="functionComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="DrawLabel" name="drawLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>721</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="functionTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая функция</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawLabel</class>
   <extends>QLabel</extends>
   <header>drawlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
